sphinx_doc_src/index.rst

```diff
@@ -1213,7 +1213,7 @@ The user can change the settings of ``fish`` by changing the values of certain v
 
 - ``fish_ambiguous_width`` controls the computed width of ambiguous-width characters. This should be set to 1 if your terminal emulator renders these characters as single-width (typical), or 2 if double-width.
 
-- ``fish_escape_delay_ms`` overrides the default timeout of 30ms after seeing an escape character before giving up on matching a key binding. See the documentation for the `bind <cmds/bind.html#special-case-the-escape-character>`__ builtin command. This delay facilitates using escape as a meta key.
+- ``fish_escape_delay_ms`` overrides the default timeout of 30ms after seeing an escape character before giving up on matching a key binding. See the documentation for the `bind <cmd-bind>`__ builtin command. This delay facilitates using escape as a meta key.
```

Comment:
This should be `cmd-bind#special-case-the-escape-character`.

Reply:
@zanchey this actually should be "cmd-bind#special-case-escape", but I didn't know that when I made PR (`special-case-the-escape-character` wasn't found anywhere near),  because of [this](fish-shell/fish-site#81) I wasn't even sure this project even interested in any kind of help. So sorry, I wouldn't be able to add changes, as I delete my fork.

Reply:
Got it. I'm going to merge with a minor fixup; I hope that's ok and thank you for your contribution.